test/style_spec.js

```diff
@@ -1,124 +1,119 @@
 import chai from 'chai';
 let assert = chai.assert;
-//import {parseLayers, walkRuleTree} from '../src/rules';
```

Comment:
We can/should still add tests for those style rules, but in the meantime I added these tests that are more about style compilation and config.